### PR TITLE
Code Quality: Update dependencies

### PR DESCRIPTION
**Pull request by hishitetsu**
**Resolved / Related Issues**
- [ ] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #issue...

**Validation**
How did you test these changes?
- [X] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### CODE REVIEW

**Comment by 0x5bfa** [comment]
@yaira2 why isn't dependabot working?

**Comment by hishitetsu** [comment]
Dependabot has tons of bugs with nuget. They are fixing bugs little by little, but it still doesn't work perfectly.

File: src/Files.App/ViewModels/UserControls/AddressToolbarViewModel.cs [hunk 1]
```diff
@@ -12,6 +12,7 @@
 using System.Windows.Input;
 using Windows.ApplicationModel.DataTransfer;
 using Windows.UI.Text;
+using FocusManager = Microsoft.UI.Xaml.Input.FocusManager;
```
[the remark this answers is not in the document]
Comment (hishitetsu) [reply src/Files.App/ViewModels/UserControls/AddressToolbarViewModel.cs:15]: Yes, updating the dependencies caused a build error here.

**Comment by yaira2** [comment]
Thank you

**Pull request merged**